src/renderer/pages/player-page.js

```diff
@@ -129,6 +129,7 @@ function renderMedia (state) {
     <MediaTagName
       src={Playlist.getCurrentLocalURL(state)}
       onDoubleClick={dispatcher('toggleFullScreen')}
+      onClick={() => { state.playing.isPaused = !state.playing.isPaused }}
```

Comment:
This should use `dispatch('playPause')`. We shouldn't modify the state from the render function, though I see that this rule is actually not being followed consistently in this file at the moment.